src/Features/VisualBasic/Portable/RemoveUnusedParametersAndValues/VisualBasicRemoveUnusedValuesCodeFixProvider.vb

```diff
@@ -52,5 +52,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedParametersAndValues
             ' VB does not support compound assignments.
             Throw New NotImplementedException()
         End Function
+
+        Protected Overrides Function GetCandidateLocalDeclarationForRemoval(declarator As VariableDeclaratorSyntax) As LocalDeclarationStatementSyntax
+            Return TryCast(declarator.Parent, LocalDeclarationStatementSyntax)
```

Comment:
Can we add a unit test to validate this code path?

Reply:
This is already hit by the existing tests, for example [this one].(https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/VisualBasicTest/RemoveUnusedParametersAndValues/RemoveUnusedValueAssignmentTests.vb#L405)

Reply:
Thanks, then it should be fine.